notebooks/20220512_initial_sophie_exploration.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>What are the <code>NA</code> here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-cf-sputum/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>